app/views/buyers/service_contracts/new.html.erb

```diff
@@ -1,8 +1,8 @@
 <h1>New service subscription</h1>
 
 <%= semantic_form_for @service_contract,
-  :url => admin_buyers_account_service_contracts_path(@account, :service_id => @service.id),
-  :html => { :'data-remote' => true } do |form| %>
+  url: admin_buyers_account_service_contracts_path(@account, service_id: @service.id),
+  html: { 'data-remote': true } do |form| %>
```

Comment:
FYI you can write `html: {data: {remote: true}}` as well
![image](https://user-images.githubusercontent.com/64276/62370515-a3a4f580-b532-11e9-8214-88e699ed03cb.png)